pkg/types/ccipocr3/common_types.go

```diff
@@ -5,8 +5,52 @@ import (
 	"fmt"
 	"math/big"
 	"strings"
+
+	libocr "github.com/smartcontractkit/libocr/offchainreporting2plus/types"
 )
 
+// UnknownAddress represents a raw address with an unknown encoding.
+type UnknownAddress []byte
+
+// NewUnknownAddressFromHex creates a new UnknownAddress from a hex string.
+func NewUnknownAddressFromHex(s string) (UnknownAddress, error) {
+	b, err := NewBytesFromString(s)
+	if err != nil {
+		return nil, err
+	}
+	return UnknownAddress(b), nil
+}
+
+// String returns the hex representation of the unknown address.
+func (a UnknownAddress) String() string {
+	return Bytes(a).String()
+}
+
+func (a UnknownAddress) MarshalJSON() ([]byte, error) {
+	return Bytes(a).MarshalJSON()
+}
+
+func (a *UnknownAddress) UnmarshalJSON(data []byte) error {
+	return (*Bytes)(a).UnmarshalJSON(data)
+}
+
+// IsZeroOrEmpty returns true if the address contains 0 bytes or if all the bytes are 0.
+func (a UnknownAddress) IsZeroOrEmpty() bool {
+	if len(a) == 0 {
+		return true // empty
+	}
+
+	for _, b := range a {
+		if b != 0 {
+			return false // zero
+		}
+	}
+	return true
+}
+
+// UnknownEncodedAddress represents an encoded address with an unknown encoding.
+type UnknownEncodedAddress libocr.Account
```

Comment:
Why define another type? Can we just use Account directly? If the package import is obnoxious, then we can use an alias:
```suggestion
type Account = libocr.Account
```

Reply:
A lot of these types are duplicated in `chainlink-ccip` so after this PR is merged, I was hoping to delete all of the duplicate types in `chainlink-ccip`. `UnknownEncodedAddress` is used heavily in chainlink-ccip in addition to being used in this CAL interface, but I guess we could separate the deletion of `UnknownEncodedAddress` from the rest of the cleanup

Reply:
If we want to preserve the name, that is fine, but an alias still seems like the simplest way to preserve compatibility :shrug:

Reply:
yeah sounds good, in interest of this whole effort of moving towards compatibility and reusability across CRE and beyond, I'll update this to `Account` here and handle the cleanup of `UnknownEncodedAddress ` in chainlink-ccip separately

Reply:
I mean this: `type UnknownEncodedAddress = libocr.Account`